cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml

```diff
@@ -41,6 +41,8 @@ data:
         add_header Cache-Control "max-age=300";
         add_header Cache-Control "must-revalidate";
         add_header ETag "{{ $.Chart.Version }}";
+        add_header 'Access-Control-Allow-Origin' * always;
```

Comment:
@AjayTripathy immediate use case for this is to do things like determining if turndown is enabled from the frontend. Given the current config, all 404s are blocked by CORS policy.